### PR TITLE
feat: punk accept bids database

### PR DESCRIPTION
**Pull request by bertmiller**
Should be merged after #148 and #147

### CODE REVIEW

File: alembic/versions/070819d86587_create_punk_snipe.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "punk_snipes",
+        sa.Column("created_at", sa.TIMESTAMP, server_default=sa.func.now()),
+        sa.Column("block_number", sa.Numeric, nullable=False),
+        sa.Column("transaction_hash", sa.String(66), nullable=False),
+        sa.Column("trace_address", sa.String(256), nullable=False),
+        sa.Column("from_address", sa.String(256), nullable=False),
+        sa.Column("punk_index", sa.Numeric, nullable=False),
+        sa.Column("min_acceptance_price", sa.Numeric, nullable=False),
+        sa.Column("acceptance_price", sa.Numeric, nullable=False),
+        sa.PrimaryKeyConstraint("transaction_hash", "trace_address"),
```
Comment (lukevs) [inline alembic/versions/070819d86587_create_punk_snipe.py:30]: throw block number up front in this key

needed to keep deletes fast

File: alembic/versions/52d75a7e0533_add_punk_bid_acceptances.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    op.create_table(
+        "punk_bid_acceptances",
+        sa.Column("created_at", sa.TIMESTAMP, server_default=sa.func.now()),
+        sa.Column("block_number", sa.Numeric, nullable=False),
+        sa.Column("transaction_hash", sa.String(66), nullable=False),
+        sa.Column("trace_address", sa.String(256), nullable=False),
+        sa.Column("from_address", sa.String(256), nullable=False),
+        sa.Column("punk_index", sa.Numeric, nullable=False),
+        sa.Column("min_price", sa.Numeric, nullable=False),
+        sa.PrimaryKeyConstraint("transaction_hash", "trace_address"),
```
Comment (lukevs) [inline alembic/versions/52d75a7e0533_add_punk_bid_acceptances.py:29]: block number up front in this key

File: alembic/versions/7cf0eeb41da0_add_punk_bids.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    op.create_table(
+        "punk_bids",
+        sa.Column("created_at", sa.TIMESTAMP, server_default=sa.func.now()),
+        sa.Column("block_number", sa.Numeric, nullable=False),
+        sa.Column("transaction_hash", sa.String(66), nullable=False),
+        sa.Column("trace_address", sa.String(256), nullable=False),
+        sa.Column("from_address", sa.String(256), nullable=False),
+        sa.Column("punk_index", sa.Numeric, nullable=False),
+        sa.Column("price", sa.Numeric, nullable=False),
+        sa.PrimaryKeyConstraint("transaction_hash", "trace_address"),
```
Comment (lukevs) [inline alembic/versions/7cf0eeb41da0_add_punk_bids.py:29]: block number up front in this key

File: alembic/versions/52d75a7e0533_add_punk_bid_acceptances.py (hunk 2 above)
Comment (bertmiller) [reply alembic/versions/52d75a7e0533_add_punk_bid_acceptances.py]: Addressed this comment on all PRs

**Review by lukevs — APPROVED** [review]

**Pull request merged**